Actions/PipelineCleanup/PipelineCleanup.ps1

```diff
@@ -1,7 +1,7 @@
 Param(
     [Parameter(HelpMessage = "Project folder", Mandatory = $false)]
     [string] $project = ".",
-    [Parameter(HelpMessage = "Specifies the parent telemetry scope for the Telemetry signal", Mandatory = $false)]
+    [Parameter(HelpMessage = "Specifies the parent telemetry scope for the telemetry signal", Mandatory = $true)]

```

Comment:
If you set it to true here, you also need to set it to true in the action.yaml file

Reply:
thinking again about it I dont believe that it should be mandatory. Ill fix it accordingly